Source/WebCore/Modules/cookie-store/CookieStoreManager.h

```diff
@@ -26,24 +26,68 @@
 #pragma once
 
 #include <wtf/Forward.h>
+#include <wtf/HashTraits.h>
 #include <wtf/RefCounted.h>
+#include <wtf/WeakPtr.h>
 
 namespace WebCore {
 
-struct CookieStoreGetOptions;
 class DeferredPromise;
+class ScriptExecutionContext;
+class ServiceWorkerRegistration;
+class WeakPtrImplWithEventTargetData;
+struct CookieStoreGetOptions;
+
+struct CookieChangeSubscription {
+    String name;
+    String url;
+
+    CookieChangeSubscription() = default;
+
+    CookieChangeSubscription(String&& name, String&& url)
+        : name(WTFMove(name))
+        , url(WTFMove(url))
+    { }
+
+    explicit CookieChangeSubscription(WTF::HashTableDeletedValueType deletedValue)
+        : name(deletedValue)
+    { }
+
+    bool operator==(const CookieChangeSubscription& other) const
```

Comment:
`= default;`
No need to write your own implementation as it does the default thing AFAICT.